src/aria/widgets/form/CheckBox.js

```diff
@@ -104,8 +96,8 @@ module.exports = Aria.classDefinition({
          */
         _checkCfgConsistency : function () {
             // for the simpleHTML case, tab index is set directly on the input
-            this._customTabIndexProvided = this._skinObj.simpleHTML;
-        },
+            this._customTabIndexProvided = true;
```

Comment:
I think we don't really need to define the `_checkCfgConsistency` method. We could move this line to the constructor, or even set the `_customTabIndexProvided` property directly in the prototype.